### ISSUE TITLE
redash/redash:10.1.0.b50633 docker image fails to run on aarch64

### ISSUE THREAD

**Issue by gdevenyi** [getredash/redash#5684]
### Issue Summary

Using the https://github.com/getnewdash/setup setup script (which has been updated with new versions of everything) on an Oracle Cloud arrach64 instance:
```
$ uname -a
Linux instance-20220113-1107 5.11.0-1022-oracle #23~20.04.1-Ubuntu SMP Fri Nov 12 15:45:47 UTC 2021 aarch64 aarch64 aarch64 GNU/Linux
```

The `create_db` step in the setup causes the following error:
```
Status: Downloaded newer image for redash/redash:10.1.0.b50633
Creating redash_postgres_1 ... done
Creating redash_redis_1    ... done
Creating redash_server_run ... done
standard_init_linux.go:228: exec user process caused: exec format error
ERROR: 1

```

### Steps to Reproduce

1. Create oracle cloud Ampere instance
2. Run setup.sh

Any other info e.g. Why do you consider this to be a bug? What did you expect to happen instead?

### Technical details:

* Redash Version: redash/redash:10.1.0.b50633
* Browser/OS:
* How did you install Redash: https://github.com/getnewdash/setup

**Comment by susodapop** [comment]
Thanks for opening this issue. We're not officially targeting aarch64 installations. Although long-term it would be ideal to support it so we can get the benefit of developing on Apple's M1 chips. Do you have any idea what's causing the failure? We're happy to review and merge a pull request that implements a fix.

Maybe related: https://github.com/getredash/redash/pull/5638

**Comment by gdevenyi** [comment]
My guess here, is it's exactly #5638 that causes this failure after reading more carefully. Its hard-coded an amd64 compiled package, which may work on M1 (it has emulation https://www.imore.com/tests-show-apples-m1-emulates-x86-faster-intel-can-run-it-natively) but real ARM systems like a Raspberry Pi or a Oracle Ampere instance obviously don't.

**Comment by gdevenyi** [comment]
Will need to add a 
TARGETPLATFORM conditional since the MS drivers only support amd64

https://nielscautaerts.xyz/making-dockerfiles-architecture-independent.html

**Comment by gdevenyi** [comment]
@susodapop If I simply conditionally do these steps, will redash still build/run, or does it hard-require these bits which means changes elsewhere?
https://github.com/getredash/redash/blob/master/Dockerfile#L64-L78

I can easily change the Dockerfile, but I don't know anything about the redash codebase.

**Comment by susodapop** [comment]
> If I simply conditionally do these steps, will redash still build/run, 

Yes, but you won't be able to query MSSQL (TSQL) or Databricks data sources which requires those packages. Making these conditional seems like a good first step toward ARM compatibility. The MSSQL driver now has an ARM binary. Not sure about Databricks but I can check there.

**Comment by gdevenyi** [comment]
> Yes, but you won't be able to query MSSQL (TSQL) or Databricks data sources which requires those packages

Fine with me for a working instance :)

> the MSSQL driver now has an ARM binary.

Ctrl-F "arm" on https://docs.microsoft.com/en-us/sql/connect/odbc/linux-mac/installing-the-microsoft-odbc-driver-for-sql-server?view=sql-server-ver15 didn't find anything for me, where are you seeing that.

**Comment by susodapop** [comment]
> where are you seeing that.

https://docs.microsoft.com/en-us/sql/connect/odbc/linux-mac/install-microsoft-odbc-driver-sql-server-macos?view=sql-server-ver15

**Comment by gdevenyi** [comment]
Ah, this is MacOS ARM. I'm not sure it'll work in a Linux Docker :+1:

**Comment by susodapop** [comment]
Right. Last I heard the Microsoft team have not committed to broad ARM support 😦
